server/handlers/postHandler.ts

```diff
@@ -35,3 +44,18 @@ export const createPostHandler: ExpressHandler<CreatePostRequest, CreatePostResp
   await db.createPost(post);
   return res.sendStatus(200);
 };
+
+export const deletePostHandler: ExpressHandler<DeletePostRequest, DeletePostResponse> = async (
+  req,
+  res
+) => {
+  if (req.body.postId === '' || req.body.postId === undefined) return res.sendStatus(404);
```

Comment:
should be a 400 since it's just a bad request. 404 means the resource wasn't found on the server.

Reply:
fixed.